### PR TITLE
Support packed attestation format without a `x5c` CBOR key

### LINKED ISSUE #150: Missing support for both "packed" formats
**Describe the bug**
The [spec](https://www.w3.org/TR/webauthn-2/#sctn-packed-attestation) says that `packed` format can be in those 2 forms:

```
packedStmtFormat = {
                           alg: COSEAlgorithmIdentifier,
                           sig: bytes,
                           x5c: [ attestnCert: bytes, * (caCert: bytes) ]
                       } //
                       {
                           alg: COSEAlgorithmIdentifier
                           sig: bytes,
                       }
```
unfortunately the code only supports the form with `x5c` and resorts to encoding empty array when no certificate chain is provided. 

**To Reproduce**
Use macbook pro touchID with Chrome - this one should produce offending attestation object.

**Expected behavior**
A `packed` attestation format in both versions should be supported/decoded. Not sure about encoding - seems like lack of `x5c` is a signal that no certificate verification is desired.

**Environment**
 - MacOS 11.6.5 (20G527)
 - Chrome Version 100.0.4896.127 (Official Build) (x86_64)
 - Version of the code: https://github.com/tweag/webauthn/releases/tag/v0.4.0.0

### PR DESCRIPTION
**Pull request by infinisil**
Should fix #150, but not tested yet.

### CODE REVIEW

**Review by ErinvanderVeen — APPROVED** [review]

File: tests/Main.hs [hunk 1]
```diff
@@ -212,12 +212,20 @@ main = Hspec.hspec $ do
         predeterminedDateTime
     it "tests whether the fixed packed register has a valid attestation" $
       registerTestFromFile
-        "tests/responses/attestation/packed-02.json"
+        "tests/responses/attestation/packed-03.json"
```
Comment (infinisil) [inline tests/Main.hs:215]: @ErinvanderVeen Why is this changed? :thinking:

Comment (ErinvanderVeen) [reply tests/Main.hs:215]: Because we tested 02 twice and 03 never, figured we should fix that :)
